Screenbox/Pages/PlayerPage.xaml

```diff
@@ -290,7 +340,7 @@
                 </Grid.RenderTransform>
                 <Image
                     x:Name="BackgroundImageNext"
-                    Source="{x:Bind AlbumArtImage.Source, Mode=OneWay}"
+                    Source="{x:Bind AlbumArtImage.ImageSource, Mode=OneWay}"
```

Comment:
This broke the background fade animation btw. You didn't change [this line](https://github.com/United600/Screenbox/blob/f0440542959b2fd0e9d5a406d2074d5ab5f66c58/Screenbox/Pages/PlayerPage.xaml.cs#L69). I can fix it if you want.

https://github.com/user-attachments/assets/e1ab35e8-c2a2-432e-ad40-28b28ccc77fc

Reply:
> This broke the background fade animation btw. You didn't change [this line](https://github.com/United600/Screenbox/blob/f0440542959b2fd0e9d5a406d2074d5ab5f66c58/Screenbox/Pages/PlayerPage.xaml.cs#L69). I can fix it if you want.

Oops, I can fix it.